scalableminds/webknossos
# Allow to restore older volume versions

This PR enables the volume version restore view and adds previewing of older volume versions.

### URL of deployed dev instance (used for testing):
- https://volumerestore.webknossos.xyz

### Steps to test:
- Open a volume/hybrid tracing. Do some volume tracing, then open the version view by clicking on "Restore older version" in the dropdown next to the save button.
- Preview older versions of the volume tracing. The segmentation data should live reload.
- Restore an older version. Then volume trace some more and save, everything should work.
- Re-Open the version view, you should see the "Reverted to version x" entry in the version log.

### Issues:
- fixes #3146 

------
